lib/PHPPdf/Core/ComplexAttribute/Background.php

```diff
@@ -196,7 +196,9 @@ private function getXCoord(Node $node, $width, $x, UnitConverter $converter)
             case self::POSITION_LEFT:
                 return $x;
             default:
-                return $x + $converter->convertUnit($converter->convertPercentageValue($this->positionX, $node->getWidth()));
+                $value = $converter->convertUnit($converter->convertPercentageValue($this->positionX, $node->getWidth()));
+                $value = str_replace(['px', 'pt', '%'], '', $value);

```

Comment:
I don't get it with the new str_replace ```str_replace(['px', 'pt', '%'], ''```. This was not there before. But actually if the resulted PDF still works in platforms I am happy.